neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
Is this really necessary? Did `/etc/os-release` gives (any) output? If this doesn't exist, there's already a last fallback.

```sh
for release_file in /etc/*-release; do
 distro+="$(< "$release_file")"
done
```